packages/client/src/products/types/productVariant.types.ts

```diff
@@ -1,6 +1,14 @@
 import type { Attribute, PurchaseChannel } from '../../types/index.js';
 import type { Price } from './price.types.js';
 
+type PublicationFeature = {
```

Comment:
This needs to be exported as well since it is being used in a type that will be exported.

Reply:
Done.